src/app/register/register.component.css

```diff
@@ -0,0 +1,10 @@
+h1{
+    text-align:center; 
+    color:darkslateblue;
+}
+div{
+    text-align:center;
+}
+span{
+    color :red;
+}
```

Comment:
better to define css for class instead of element/tag and use that class in html.